dune_daq_services/runservices/runregistry-rest/runregistry.yml

```diff
@@ -26,43 +28,66 @@ spec:
            - matchExpressions:
              - key: {{ kubernetes_label.worker }}
                operator: Exists
+      initContainers:
+      - name: wait-for-run-postgres
+        image: busybox:1.28
+        command: ['sh','-c', 'until nc -z runservices-postgresql.runservices.svc.cluster.local 5432; do echo "Waiting for PostgreSQL"; sleep 2; done']
       containers:
-      - image: ghcr.io/dune-daq/pocket-runregistry-rest:0.8 # this tag is different than standard
```

Comment:
I'm pretty sure some version of this line is required.

Reply:
Its not required I thought this became obsolete with the unified microservices image. I've tested and everything still runs as expected

Reply:
Or at least that's how I've understood it